src/components/Product/ListProduct.js

```diff
@@ -53,33 +54,59 @@ export default function ListProduct() {
                   <th>Name</th>
                   <th>Date</th>
                   <th>Last Date</th>
+                  <th>Estimate</th>
                 </tr>
               </thead>
               <tbody>
-                {data.map((value, key) => {
-                  const differenceDays =
-                    (currentDateSeconds -
-                      (!!value.lastPurchasedDate &&
-                      !!value.lastPurchasedDate.seconds
-                        ? value.lastPurchasedDate.seconds
-                        : 0)) /
-                    (3600 * 24);
-                  return (
-                    <tr key={key}>
-                      <td>
-                        <input
-                          type="checkbox"
-                          checked={differenceDays <= 1}
-                          id={value.name}
-                          onChange={() => select(value)}
-                        />
-                        {value.name}
-                      </td>
-                      <td>{value.date} </td>
-                      <td></td>
-                    </tr>
-                  );
-                })}
+                {data
+                  .sort(function (a, b) {
+                    if (a.name > b.name) {
+                      return 1;
+                    }
+                    if (a.name < b.name) {
+                      return -1;
+                    }
+                    return 0;
+                  })
```

Comment:
This code should be explained a bit more.

Please, add a comment for each `if` statement to understand what's happening. You could also, move that to another function to improve readability!